cluster/manifests/stackset-controller/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-{{ $version := "v1.4.41" }}
+{{ $version := "v1.4.44" }}
```

Comment:
Shouldn't this also be `1.4.45` like the rest below? Or is the image version different from the module version?

Reply:
Right :facepalm: updated.